README.md

```diff
@@ -36,6 +36,33 @@ Cons: Only includes standard libraries - no Debian `-dev` packages.
 
 [MentorGraphics toolchain](http://sourcery.mentor.com/public/gnu_toolchain/arm-none-linux-gnueabi/arm-2014.05-29-arm-none-linux-gnueabi.exe) (formerly known as CodeSourcery).
 
+### Windows 10
+
+Windows 10 supports the Ubuntu Subsystem for Windows which allows us to install and execute the entire compiler toolchain. The steps required to compile native-mode EV3 applications on Windows 10 is as follows:
```

Comment:
There's a mention at the beginning of the line too: "Ubuntu Subsystem for Windows". When I see a Capitalized Set of Words, I assume it to be a proper name, which could give the wrong impression because this isn't the official term. It's minor, but I figured I would point it out.